src/components/ComponentDetails/PropTable/index.tsx

```diff
@@ -53,6 +54,17 @@ const PropTable: React.FC<PropTableProps> = ({ propData, typeLibrary }) => {
     []
   );
 
+  const hasObjectDefaultValue = useMemo(
+    () =>
+      propData.some(
+        ({ default: defaultValue }) =>
+          typeof defaultValue === "string" &&
+          defaultValue.trim().startsWith("{") &&
+          defaultValue.trim().endsWith("}")
```

Comment:
Just a tiny thing - might it be worth putting this logic into a function and using it on lines 103 and 122 as well to reduce duplication code?